pkg/controllers/webui/resource.go

```diff
@@ -58,8 +58,7 @@ func syncPodSpec(wi *v1alpha1.WebUI, dp *appsv1.Deployment) {
 }
 
 func getRollingUpdateStrategy(wi *v1alpha1.WebUI) *appsv1.RollingUpdateDeployment {
```

Comment:
I think we should simply return nil if UpdateStrategy is not specified and ask kruise to set defaults